### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Sid72020123/scratchconnect#44] [Q&A] [answered]
https://github.com/Sid72020123/scratchconnect/blob/b312d55764553826d778866f4c1ed7ef0cff33cb/scratchconnect/User.py#L53

Wouldn't simply 
```python
if not self._logged_in:
    #do stuff
```
work?

**Comment by aidaner** [comment, marked as answer]
no because if you do it that way then your checking if it’s true instead of false

**Reply by Quantum-Codes** [reply]
Oops I didn't put a not, updated post

**Reply by Sid72020123** [reply]
It should work. But I won't change that. Anyways both the methods work.
